release-notes/1.20.0.md

```diff
@@ -0,0 +1,63 @@
+* The included Mesa driver is tracking the latest 23.2.1 version and the `virglrenderer` is also tracking the latest upstream.<!--AC-1933-->
+* Some improvements to the swagger response for the AMS HTTP API are included. <!--AC-1923-->
+
+### Streaming improvements
+
+* The required NVIDIA driver version has been updated from the 525 series to 535 series.<!--AC-1924-->
+* Alpha quality support for Vulkan on NVIDIA GPUs is now available through a feature flag. To use this feature, you require newer NVIDIA drivers (versions 545 or later). Since the feature is at Alpha level, bugs and crashes can occur. If you are interested in early testing, contact us through the [users forum on discourse](https://discourse.ubuntu.com/c/anbox-cloud/users/148). Stable product support for this feature is planned for the 1.21 release.
+
+### Dashboard improvements
+
+With the 1.20.0 release, we have made the following improvements to the web dashboard experience:
+
+* Improvements to session streaming statistics
+* Ability to select and delete multiple instances
+* Ability to change the Anbox Cloud configuration using the web dashboard
+* Simplified app version selection for new sessions
+* Improvements to general design and layout
+* Warning messages to improve user experience
+
+### Other
+
+* Android security updates for November 2023 (see [Android Security Bulletin - November 2023](https://source.android.com/docs/security/bulletin/2023-11-01) for more information).
+* The Android WebView has been updated to [119.0.6045.66](https://chromereleases.googleblog.com/2023/10/early-stable-update-for-android_01005299231.html).
+* We have modernised the existing NATS charm to align with best practices in developing charms. The new NATS charm is available at https://charmhub.io/nats. The 1.21.0 release of Anbox Cloud will automatically upgrade to the new charm.
+
+## Deprecations
+
+* The [`instance-type` attribute](https://discourse.ubuntu.com/t/24197#instance-type) is deprecated now. You can use the `resources` attribute to specify any custom resource requirements.
+
+  Also, in the Anbox Cloud documentation, the use of the term *Instance type* to indicate a set of resources is deprecated and will be completely removed in the future releases. Instead, the term *Resource preset* is used to indicate a set of resources available for an instance.
+
+## Removed functionality
+
+* The Mesa build in `/opt/mesa` is removed. <!--AC-1928-->
+* The Grafana and Prometheus tools for monitoring are removed. The Anbox Cloud roadmap has plans of providing a monitoring solution using the [Canonical Observability Stack](https://charmhub.io/topics/canonical-observability-stack). <!--AC-1925-->
+
+## Bug fixes
+The following bugs are fixes in the Anbox Cloud 1.20 version:
+
+* Permission errors when Android tries to access cgroup. <!--AC-1748-->
+* Missing support for VPU slots for `amc` commands. <!--AC-2004-->
+
```

Comment:
* Anbox IME remaining popped up when streaming from a tablet

Reply:
The first three are added as bug fixes and the last one is added as a core stack improvement (as discussed)